spec/support/proxies.rb

```diff
@@ -1,6 +1,7 @@
 module ProxiesHelper
   def create_proxy_using_partial(*users)
     users.each do |user|
+      sleep(1)
```

Comment:
Is this because of those tab fade-in animations? In other projects, I think we've had a way to disable them globally; maybe @mejackreed remembers off-hand?